x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connector_detail/components/generated_config_fields.tsx

```diff
@@ -121,6 +121,8 @@ export const GeneratedConfigFields: React.FC<GeneratedConfigFieldsProps> = ({
           </EuiFlexItem>
           <EuiFlexItem>
             <EuiLinkTo
+              external
```

Comment:
I don't think this is a good idea, to be honest. We should not be forcing people into opening new windows, this is [not a good practice from an accessibility standpoint](https://www.w3.org/TR/WCAG20-TECHS/G200.html), and it doesn't fit Kibana's general UX.

Reply:
I think "maintaining the user's current location and progress" is a solid enough reason to open these links in a new window.

Reply:
Actually regarding this topic IMO, every time we take users from Search solution to Platform management solution and viceversa should be using an external link. Users get confused when we take them to an almost complete different internal sub-product, changing the main navigation and layout.